.github/ci_templates/helper_jobs.yml

```diff
@@ -80,15 +89,21 @@ git_info:
             githubBranch  = GITHUB_REF_TYPE == "branch" ? refName : ""
             githubSHA     = context.sha
             core.info(`${context.eventName} event: set git info from context: ${JSON.stringify({GITHUB_REF_NAME, GITHUB_REF_TYPE, sha: context.sha })}`)
+            if (context.issue && context.issue.number) {
+              prNumber = context.issue.number
+            }
           }
 
+
```

Comment:
It is better to use core.notice to report to user about detected PR number and event after this long `if`.